docs/services/navigation/_index.md

```diff
@@ -115,6 +119,56 @@ The following attributes are available for `Navigation` services:
 | `meters_per_sec` | float | Optional | The default linear velocity for the [base](/components/base/) in meters per second. <br> Default: `0.3` |
 | `obstacles` | obj | Optional | Any obstacles you wish to add to the robot's path. See the [Motion Service](/services/motion/) for more information. |
 
+### Configure and calibrate the Frame System Service for GPS Navigation
+
+{{% alert title="Info" color="info" %}}
+
+The [Frame System Service](/services/frame-system/) is an internally managed and mostly static system for storing the reference frame of each component of a robot within a coordinate system configured by the user.
+
+It stores the required contextual information for Viam's services like [Motion](/services/motion/) and [Vision](/services/vision/) to use the position and orientation readings returned by components like [movement sensors](/components/movement-sensor/).
+
+{{% /alert %}}
+
+For the most accurate autonomous GPS navigation with the Navigation Service, configure and calibrate the Frame System Service for the components of your robot.
```

Comment:
```suggestion
To make sure your rover base's autonomous GPS navigation with the Navigation Service is accurate, configure and calibrate the Frame System Service for the components of your robot.
```

Reply:
@JessamyT @tessavitabile Do you think this is sufficient to make that clear? I could also add an intro sentence or tip elsewhere on the page

Reply:
LGTM!

Reply:
I see this is already merged now, but I think something more similar to your original sentence structure is easier to parse in this case @sguequierre 
```suggestion
To ensure the accuracy of your rover base's autonomous GPS navigation with the Navigation Service, configure and calibrate the Frame System Service for the components of your robot.
```
or some other restructuring. Not a high priority to fix now that it's merged, perhaps, but the subject of this clause is so long that as a reader, I find myself trying to find a verb long before the verb appears in the sentence.